charts/s1-agent/templates/agent/daemonset.yaml

```diff
@@ -76,6 +78,10 @@ spec:
           - name: persistence
             mountPath: {{ include "persistentDir" . }}
 {{- end }}
+          - name: config
+            mountPath: /opt/configmaps/config/config.yaml
```

Comment:
Maybe put the `/opt/configmaps/config/config.yaml` path in values.yaml and use `.Values.agent_config_map` here?

Reply:
The mount path is in the agent, so I don't think it is something that we want customers to control

Reply:
OK, didn't know customers can change it, I just thought maybe we can reduce writing it twice